### DISCUSSION TITLE
v1.6.0 🎉

### DISCUSSION THREAD

**Discussion by KevinMGranger** [dora-metrics/pelorus#493] [General]
## 🚀 Features

- Allow JIRA to define custom resolution field and search query @mpryc (#466)
- Add a github failure exporter @weshayutin (#457)
- Use ConfigMap as means for configuration @mpryc (#445)
- Fix JIRA query string to accept project FIRST and allow more then one project @mpryc (#453)
- Fix issue #448 to make handling of shell env vars consistently @mpryc (#449)
- More commit debug logs, exit Build inspection early if not yet ready @KevinMGranger (#418)
- Extract image SHAs from status.containerStatuses as well as spec.containers @KevinMGranger (#395)

## 🐛 Bug Fixes

- get ALL github issues, open and closed @weshayutin (#489)
- Ensure values from secrets has higher precedence than from configmap @mpryc (#469)
- Fix Issue #446 where committime exporter had to be restarted @mpryc (#461)
- Fix for #304 to allow JIRA obtain more then 50 results @mpryc (#427)
- Fix issue #278 - failure pod crashing @mpryc (#425)

## 🛠️ Misc. Improvements

- Prow ci script @mpryc (#472)
- Sample app todo @weshayutin (#442)
- Make exporters log version information on startup @KevinMGranger (#441)
- Fix issue #430 to allow running Mockoon container without SELinux @mpryc (#431)
- Github action to use Makefile for conf-tests @mpryc (#415)
- Easy release process @mpryc (#401)
- OWNERS file required by OpenShift workflows @mpryc (#400)

## 📖 Documentation

- Example secrets that can be applied to the cluster @mpryc (#470)
- Remove duplicate charts values.yaml file @weshayutin (#405)
- add a 'make help', 'make list' action @weshayutin (#413)
- Cleanup and documentation of the python requirements @mpryc (#406)

## 🔡 Other changes

- Minor release of Pelorus 1.6.0 @mpryc (#483)
- Make default values.yaml use default configmaps @KevinMGranger (#485)
- Use PULL\_NUMBER in e2e tests scripts @mpryc (#487)
- Ensure curl backwards compatibility in e2e tests @mpryc (#484)
- sample needs webhook secret and proper app path @weshayutin (#432)
- Fix for issue #428 and some minor Makefile nits @mpryc (#429)
- Github actions should use Makefile @mpryc (#412)
- Better missing attribute errors @KevinMGranger (#411)
- Make chart-lint github action trigger on changes to requirements-dev.txt @KevinMGranger (#408)
- Add test target to makefile @KevinMGranger (#410)
- Disable fail-fast on all python github actions @KevinMGranger (#402)
- Remove python patch version check @KevinMGranger (#404)

## 📦 Operator Updates

- Tag latest released version with 'stable' tag. @mpryc (#481)
- Add readiness and liveness probes, associate exporter type label @mpryc (#467)
- Use new python openshift version 0.13.1 @mpryc (#460)
- Test to cover JIRA token and password authentication. @mpryc (#424)
- Improves workflow for building Pelorus images @mpryc (#407)

**Full Changelog**: https://github.com/konveyor/pelorus/compare/v1.5.0...v1.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/konveyor/pelorus/releases/tag/v1.6.0'>v1.6.0 🎉</a>.</em>

**Comment by mateusoliveira43** [comment]
Closing as we had other releases since this one and we agreed to inform new releases on  github releases, slack channels and google groups
